index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="container">
```

Comment:
- [ ] Please consider changing this **div** to a more semantic tag, like:

- main
- section

Reply:
Please, note that your comment should be descriptive and clear.

As I mentioned above, Kindly, I suggest you use polite and clear comments, like:

```
This is a polite suggestion: "I think it would be better to use semantic tags"

This reads as an order: "Use semantic tags"
```

Reply:
Great!

Reply:
[ OPTIONAL ]
- [ ] Please consider using emojis in your comments as it makes your comments human friendly 👽 😄